packages/braid-design-system/src/lib/components/Accordion/AccordionItem.css.ts

```diff
@@ -1,8 +1,9 @@
 import { style } from '@vanilla-extract/css';
 import { calc } from '@vanilla-extract/css-utils';
 import { vars } from '../../themes/vars.css';
+import { virtualTouchable } from '../private/touchable/virtualTouchable.css';
 
-export const button = style({});
+export const button = style([{}, virtualTouchable]);
```

Comment:
Any reason not to just re-export the classname?
```suggestion
export const button = virtualTouchable;
```

Reply:
Yep need to create a new reference to be used in the following selectors